libs/island-ui/core/src/lib/DialogPrompt/DialogPrompt.tsx

```diff
@@ -89,60 +89,70 @@ export const DialogPrompt = ({
         }
         return (
           <Box
-            position="relative"
-            marginX={3}
-            paddingX={[1, 4, 4, 8]}
-            paddingY={[6, 6, 6, 12]}
-            borderRadius="large"
             background="white"
-            className={styles.content}
+            paddingY={[3, 6, 12]}
+            paddingX={[3, 6, 12, 15]}
           >
-            <GridContainer position="none">
-              <FocusableBox
-                component="button"
-                onClick={handleClose}
-                className={styles.close}
-              >
-                <Icon icon="close" color="blue400" size="medium" />
-              </FocusableBox>
-              <Text variant="h2" as="h3" paddingBottom={2}>
-                {title}
-              </Text>
-              {description && (
-                <Text variant="intro" paddingBottom={2}>
-                  {description}
+            <Box className={styles.closeButton}>
+              <Button
```

Comment:
For better accessibility, we should add an aria label since this button has no text.